controllers/mandatees-decisions.ts

```diff
@@ -1,198 +1,89 @@
 import { PUBLICATION_STATUS } from '../util/constants';
+import { copyBeleidsdomeinInfo } from './mandataris-besluit/beleidsdomein';
+import { copyFractionInfo } from './mandataris-besluit/fractie';
+import { copyMandatarisInfo } from './mandataris-besluit/mandataris';
+import { copyPersonInfo } from './mandataris-besluit/persoon';
 
 export async function processMandatarisForDecisions(
-  mandatarisSubject: Term,
+  mandatarisSubject: string,
 ): Promise<void> {
-  const isMandataris = await isSubjectOfType(
-    TERM_MANDATARIS_TYPE,
-    mandatarisSubject,
-  );
-  if (!isMandataris) {
-    console.log(
-      `|> URI: ${mandatarisSubject.value} is not of type: ${TERM_MANDATARIS_TYPE.value}`,
-    );
-    return;
-  }
-
-  // The decision is actually a besluit:Artikel this
-  // because the besluit does not have a direct relation to the mandataris yet
-  const decision = await findDecisionForMandataris(mandatarisSubject);
-  if (!decision) {
-    console.log(
-      `|> Could not find a decision for mandataris: ${mandatarisSubject.value}`,
-    );
-    mandatarisQueue.addToManualQueue(mandatarisSubject);
+  const { valid, besluitUri, type } =
+    await isValidMandataris(mandatarisSubject);
+  if (!valid || !besluitUri) {
     return;
   }
-
-  await handleTriplesForMandatarisSubject(mandatarisSubject);
+  const mandatarisPointer: MandatarisBesluitLookup = {
+    mandatarisUri: mandatarisSubject,
+    besluitUri,
+    type,
+  };
+  await handleMandatarisSubject(mandatarisPointer);
   await updatePublicationStatusOfMandataris(
     mandatarisSubject,
     PUBLICATION_STATUS.BEKRACHTIGD,
   );
 }
 
-export async function handleTriplesForMandatarisSubject(
-  mandatarisSubject: Term,
-) {
-  console.log(`|> Mandataris uri: ${mandatarisSubject.value}`);
-  const isExitingInLmbDatabase =
-    await isMandatarisInLmbDatabase(mandatarisSubject);
-  console.log(
-    `|> Mandataris exists in LMB database? ${isExitingInLmbDatabase}`,
-  );
-
-  const mandaat = await findMandateOfMandataris(mandatarisSubject);
-  console.log('|> Mandaat for mandataris', mandaat?.value);
-  if (!mandaat) {
-    console.log(
-      `|> No mandaat found for mandataris with subject: ${mandatarisSubject.value} \n|>\n`,
-    );
-    mandatarisQueue.addToManualQueue(mandatarisSubject);
-    return;
-  }
-
-  const mandatarisGraph = await findBestuurseenheidForMandaat(mandaat);
-  console.log(`|> mandataris graph: ${mandatarisGraph?.value ?? undefined}.`);
-
-  if (!mandatarisGraph) {
-    console.log(
-      `|> Could not find graph from mandaat: ${mandaat.value}. Continueing to the next subject.\n|>\n`,
-    );
-    mandatarisQueue.addToManualQueue(mandatarisSubject);
-    return;
-  }
-
-  const incomingTriples = await getTriplesOfSubject(
-    mandatarisSubject,
-    TERM_STAGING_GRAPH,
-  );
-  console.log(
-    `|> Found ${incomingTriples.length} triples in the staging graph for mandataris.`,
+async function isValidMandataris(mandataris: string) {
```

Comment:
Function name is not really descriptive. It does more than it says. It checks whether the madataris is valid and whether it has a besluit. Maybe you could call it `isValidMandatarisWithBesluit` ?